packages/daimo-api/src/db/db.ts

```diff
@@ -50,6 +51,8 @@ export class DB {
             max_uses INT NOT NULL DEFAULT 1
           );
           ALTER TABLE invitecode ADD COLUMN IF NOT EXISTS zupass_email VARCHAR DEFAULT NULL;
+          ALTER TABLE invitecode ADD COLUMN IF NOT EXISTS inviter CHAR(42) DEFAULT NULL;
+          ALTER TABLE invitecode ADD COLUMN IF NOT EXISTS bonus_cents INT DEFAULT 0 NOT NULL;
```

Comment:
bonus_cents is a kinda enigmatic name. is this a hypothetical future per-signup bonus for the inviter?

Reply:
this is used right now as the denomination to send to invitee and the inviter from us. let me know what you’d call it, happy to change.

Reply:
> maybe split it out into `inviter_bonus_cents` / `invitee_bonus_cents` ?

done